mono/utils/mono-tls.h

```diff
@@ -36,6 +36,19 @@ typedef enum {
 
 #include <windows.h>
 
+/*
+* These APIs were added back in Windows SDK 14393. Let's redirect them to
+* Fls* APIs on older SDKs just like Windows 8.1 headers do
+*/
+#if G_HAVE_API_SUPPORT(HAVE_UWP_WINAPI_SUPPORT)
+#if WINDOWS_SDK_BUILD_VERSION < 14393
+#define TlsAlloc() FlsAlloc(NULL)
+#define TlsGetValue FlsGetValue
```

Comment:
We can probably do this unconditionally. The APIs date to Server 2003. Or are there old headers still around?

Reply:
This is needed on Windows 10 SDKs 10240 and 10586, or this doesn't compile. And these SDKs are from 2015 :).

Reply:
Ah, Win32 fragmentation. What if we declare the functions ourselves?

Reply:
Then we get linker errors.

Reply:
Craziness, alas. Ok we'll fix it much later.